homeassistant/components/ecobee/manifest.json

```diff
@@ -9,7 +9,7 @@
   },
   "iot_class": "cloud_polling",
   "loggers": ["pyecobee"],
-  "requirements": ["python-ecobee-api==0.2.14"],
+  "requirements": ["python-ecobee-api==0.2.17"],
```

Comment:
This PR does two things:
- bumping the dependency
- adding the new service

The two things can be split, so they should be split into two PRs.
Please move the dependency bump into a new PR. Leave this PR in draft until the other one is merged.

Reply:
Have removed from this pr and added to https://github.com/home-assistant/core/pull/102900